lib/phoenix/live_dashboard/system_info.ex

```diff
@@ -235,10 +240,75 @@ defmodule Phoenix.LiveDashboard.SystemInfo do
     end
   end
 
+  ## Socket callbacks
+
+  def sockets_callback(search, sort_by, sort_dir, limit) do
+    sorter = if sort_dir == :asc, do: &<=/2, else: &>=/2
```

Comment:
Please see how we do sorting in the other callbacks, you can reuse the same logic.

Reply:
We saw it, but it won't work on strings (ie. local or foreign address). Or are we missing something?

Reply:
Ah, good point. Ignore this then!